CIPs/0008.md

```diff
@@ -38,24 +39,22 @@ This http address should be referred to as the root. The expectation here is tha
 Here are some common ways of modeling applications into this format:
 
 - For transaction metadata, the account can encrypt the metadata and put it in a file with the transaction hash as the metadata. Further CIPs could specify a standard for thee structure of that metadata.
-    - `dataPath`: `/transactions/${transactionHash}`
-    - file contents: Encrypted, structured transaction metadata (comments, or otherwise)
+  - `dataPath`: `/transactions/${transactionHash}`
+  - file contents: Encrypted, structured transaction metadata (comments, or otherwise)
 - Data in relation to other accounts could be communicated to other accounts with:
-    - `dataPath`: `/accounts/${otherAccountAddress}`
-    - file contents: Structured data such as:
-        - `sessionKeys`: The session key(s) encrypted with dataEncryptionKey of `otherAccount` on `Accounts.sol`
-        - `attestations`: These could be signed data attestating things such an audit or badges. An attested account could also store the attestations from the attestor here, as a client can just verify the signature of the attestor.
+  - `dataPath`: `/accounts/${otherAccountAddress}`
+  - file contents: Structured data such as:
+    - `sessionKeys`: The session key(s) encrypted with dataEncryptionKey of `otherAccount` on `Accounts.sol`
+    - `attestations`: These could be signed data attestating things such an audit or badges. An attested account could also store the attestations from the attestor here, as a client can just verify the signature of the attestor.
 - Profile information could be stored on this system with:
-    - `dataPath`: `/profile/name` or `/profile/picture`
-    - file contents: the data unencrypted, or encrypted to session keys available to the recipients
+  - `dataPath`: `/profile/name` or `/profile/picture`
+  - file contents: the data unencrypted, or encrypted to session keys available to the recipients
 - Releases:
-    - `dataPath`: `/releases/${hashOfRelease}`
-    - file contents: The signature of the key on the hash of the release
+  - `dataPath`: `/releases/${hashOfRelease}`
+  - file contents: The signature of the key on the hash of the release
 
 If no assocation between to the `dataPath` should be leaked, the `dataPath` itself could be encrypted. The recipient would then just have to decrypt all `dataPath`s (which could be available in an index file).
```

Comment:
Is this still going to be the recommendation? With the "indirect" encryption technique, the only one we are shipping in version 1 of the implementation, this is indeed still a concern. In the "direct" encryption technique, no semantic information about the data is leaked through a plaintext data path. So an updated suggestion here may be to use the "direct" encryption strategy for cases where the dataPath itself leaks non-public information. (Or to think of a way to about including the information in the data path)